src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,5 @@
+.footer {
+  height: 30px;
+  width: 100%;
+  background: #f1f1f1;
```

Comment:
I would add explicit color values in a separate css that would be project specific and reference them here.

Reply:
Postponing this task to do when we are adjusting the UI